docs/reference/index.md

```diff
@@ -1,31 +1,22 @@
 # Reference
 
-This is our reference section. It serves to provide general reference
-information about {{brand}} services.
+This is our reference section. It serves to provide general reference information about {{brand}} services.
 
 ## Feature and service availability
 
-* Our **[Feature Support Matrix](features/index.md)** lists cloud
-  features and their availability across {{brand}}
-  regions.
+* Our **[Feature Support Matrix](features/index.md)** lists cloud features and their availability across {{brand}} regions.
 
 * The **[Limitations](limitations/index.md)** section lists support limitations for specific {{brand}} services.
 
-* The **[Flavors](flavors/index.md)** reference explains our naming
-  convention for pre-defined CPU/RAM/disk configurations ("flavors")
-  for server instances, and their availability across
-  {{brand}} regions.
+* The **[Flavors](flavors/index.md)** reference explains our naming convention for pre-defined CPU/RAM/disk configurations ("flavors") for server instances, and their availability across {{brand}} regions.
 
-* Our **[Service Version Matrix](versions/index.md)** lists the
-  open source software versions running in our {{brand}}
-  regions.
+* The **[Volume Types](volumes/index.md)** reference lists the available persistent block storage ("volume") types.
+
+* Our **[Service Version Matrix](versions/index.md)** lists the open source software versions running in our {{brand}} regions.
 
 ## API reference
 
-* The **[API reference](api/index.md)** documentation provides useful
-  information for accessing Application Programming Interfaces (APIs),
-  getting information, and manipulating all sorts of {{brand}}
-  objects.
+* The **[API reference](api/index.md)** documentation provides useful information for accessing Application Programming Interfaces (APIs), getting information, and manipulating all sorts of {{brand}}  objects.
```

Comment:
There is this double-space after {{brand}}...

Reply:
Good catch, thanks. Fixed.